src/views/errors/error.njk

```diff
@@ -1,4 +1,16 @@
 <!-- Found in common folder -->
 {% extends "unrecoverable-error.njk" %}
+{% from "components/onPageLoad/macro.njk" import ga4OnPageLoad %}
```

Comment:
I think the package has a component we can import rather than having to define it ourselves - see the README https://www.npmjs.com/package/@govuk-one-login/frontend-analytics
> A Nunjuck component can be used for a reusable solution. The ga4-opl component, available in the "components" folder, lets you add Page view tracking code with just one line of code.